src/app/COMPONENTS/task-details/task-details.component.ts

```diff
@@ -56,10 +56,10 @@ export class TaskDetailsComponent implements OnInit {
     })
   }
 
-  updateTaskDetails() {
+  updateTaskDetails(): Subscription | null {
```

Comment:
you also don't need to return anything here the type for these functions where nothing is return is void